scripts/prow/cli-integration.sh

```diff
@@ -0,0 +1,11 @@
+set -e
+apk add nodejs npm
+install_dir="/usr/local/bin"
+mkdir -p "$install_dir"
+pushd "$install_dir" || exit
+curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && tar -zxvf kyma.tar.gz && chmod +x kyma && rm -f kyma.tar.gz
+kyma version --client
+popd || exit
+kyma provision k3d --ci
+kyma deploy --ci --concurrency=8 --profile=evaluation --source=main
+make -C "../../kyma-project/kyma/tests/fast-integration" "ci"
```

Comment:
```suggestion
make -C "../kyma/tests/fast-integration" "ci"
```

Reply:
the reconciler is in the Kyma-incubator not Kyma-project that's why we need this

Reply:
Oh yeah, correct. Then ignore that suggestion